Decred Wallet/es.lproj/Localizable.strings

```diff
@@ -395,14 +395,17 @@
 "forMoreInformationPleaseVisit" = "Para más información por favor visita";
 
 /* Settings */
-"spendingPinPassInfo" = "Requerido para enviar fondos.";
-"startupPinPass" = "PIN/Contraseña de inicio";
-"changeStatupPinPass" = "Cambiar PIN/Contraseña de inicio";
-"startupPinPassInfo" = "Requerido para ingresar a la app.";
-"spendUnconfirmedFund" = "Spend Unconfirmed Funds";
+"spendingPinPassInfo" = "Required to send funds.";
+"startupPinPass" = "Startup PIN/Password";
+"changeStatupPinPass" = "Change startup PIN/Password";
+"startupPinPassInfo" = "Required to enter app.";
+"spendUnconfirmedFund" = "Spend unconfirmed funds";
 "incomingTxNotification" = "Notificaciones de transacciones entrantes";
 "currencyConversion" = "Conversión actual";
-"connectIpDesc" = "Conectado al par";
+"connectIpDesc" = "Connect to specific peer";
```

Comment:
This can remain as is, I think. There's no significant change, users in this language will still get the intended meaning.

Reply:
strings should be uniform, it might have a better explanations in other languages